units/crons.go

```diff
@@ -186,3 +186,11 @@ func PopulateHostMonitoring(env evergreen.Environment) amboy.QueueOperation {
 		return catcher.Resolve()
 	}
 }
+
+func EventMetaJobQueueOperation() amboy.QueueOperation {
+	return func(q amboy.Queue) error {
+		err := q.Put(NewEventMetaJob(q))
```

Comment:
I don't think the id of these will collide in the proper way. 

I typically run the periodic job 2x as often as the actual interval based on the part-of-unit-time mechanism